production/helm/loki/values.yaml

```
@@ -62,13 +62,17 @@ deploymentMode: SimpleScalable
 # -- Configuration for running Loki
 # @default -- See values.yaml
 loki:
+  # Configures the liveness probe for all of the Loki pods
+  livenessProbe: null
   # Configures the readiness probe for all of the Loki pods
   readinessProbe:
     httpGet:
       path: /ready
       port: http-metrics
     initialDelaySeconds: 30
     timeoutSeconds: 1
+  # Configures the startup probe for all of the Loki pods
+  startupProbe: null
```

```suggestion
  startupProbe: {}
```


See https://github.com/grafana/loki/pull/18551#issuecomment-3112597924